tests/LANraragi/Plugin/Metadata/Fakku.t

```diff
@@ -0,0 +1,56 @@
+use strict;
+use warnings;
+use utf8;
+use Data::Dumper;
+
+use Cwd qw( getcwd );
+use Mojo::JSON qw(decode_json encode_json);
+use Mojo::File;
+use Mojolicious;
+use LANraragi::Model::Config;
+
+use Test::More;
+use Test::Deep;
+use Test::MockObject;
+
+my $cwd = getcwd();
+my $SAMPLES = "$cwd/tests/samples";
+
+my $mock_log = Test::MockObject->new();
+$mock_log->mock( 'debug', sub { } );
+$mock_log->mock( 'info', sub { } );
+
+my @tags_list_from_gallery = (
+    'Artist:Hamao', 'Parody:Original Work', 'Magazine:Comic Kairakuten 2020-04', 'Publisher:FAKKU',
+    'Language:English', 'color', 'schoolgirl outfit', 'osananajimi', 'unlimited', 'non-h', 'illustration'
+);
+
+
+use_ok('LANraragi::Plugin::Metadata::Fakku');
+
+note("testing searching URL by title ...");
+
+{
+    my $html = ( Mojo::File->new("$SAMPLES/fakku/001_search_response.json")->slurp );
```

Comment:
This is an HTML sample, but the file extension is .json.

Reply:
Yep!